### PR TITLE
fix(storefront): BCTHEME-395 Wish List drop down is truncated on product page

### PR DESCRIPTION
**Pull request by yurytut1993**
#### What?

This PR has fix for truncated wish list dropdown

#### Tickets / Documentation

[BCTHEME-395](https://jira.bigcommerce.com/browse/BCTHEME-395)

#### Screenshots (if appropriate)

wishlist

### CODE REVIEW

File: assets/scss/components/stencil/productView/_productView.scss [hunk 1]
```diff
@@ -119,17 +119,9 @@
...
-// 1. for Androind Chrome and IE with zoom horizontal scroll fix
-//
-// -----------------------------------------------------------------------------
 
 .productView-details {
     padding-bottom: spacing("single") + spacing("third");
-
-    &.product-options {
-        overflow: hidden; // 1
-    }
```
Comment (BC-tymurbiedukhin) [inline assets/scss/components/stencil/productView/_productView.scss:132]: Could you please verify that this issue is not returned?

File: assets/scss/layouts/products/_productSwatch.scss [hunk 2]
```diff
@@ -45,6 +44,13 @@ $second_value : str-slice($value_of_swatch_size, $position_of_x + 1);
         transform: translate(-50%, 0);
         visibility: visible;
         z-index: 5000;
+        padding: 3px;
+        border: 1px solid stencilColor("input-border-color-active");
+
+    }
+    .form-option-image {
```
Comment (BC-tymurbiedukhin) [inline assets/scss/layouts/products/_productSwatch.scss:51]: Why are we changing swatch styles in context of this task?

File: assets/scss/components/stencil/productView/_productView.scss (hunk 1 above)
Comment (yurytut1993) [reply assets/scss/components/stencil/productView/_productView.scss:132]: Confirming

File: assets/scss/layouts/products/_productSwatch.scss (hunk 2 above)
Comment (yurytut1993) [reply assets/scss/layouts/products/_productSwatch.scss:51]: It allows to remove overflow that cause truncation of wishlist

**Review by bc-alexsaiannyi — APPROVED** [review]

**Comment by yurytut1993** [comment]
@BC-tymurbiedukhin  @bc-alexsaiannyi 
this how swatch options were looking before - even when they inactive they take place
swatch-options

**Comment by BC-tymurbiedukhin** [comment]
> @BC-tymurbiedukhin @bc-alexsaiannyi
> this how swatch options were looking before - even when they inactive they take place

How does it look like currently?

**Comment by yurytut1993** [comment]
> > @BC-tymurbiedukhin @bc-alexsaiannyi
> > this how swatch options were looking before - even when they inactive they take place
> 
> How does it look like currently?

currently it is truncated. in case of just removing overflow horizontal scroll will be permanent
https://user-images.githubusercontent.com/66325265/110444711-06bf1c80-80c6-11eb-9131-8294f0a4bdf3.mov

after fix it will appear only in case of hover on necessary swatch item
https://user-images.githubusercontent.com/66325265/110444813-1d657380-80c6-11eb-9bcb-2cf7c14a5358.mov

**Comment by yurytut1993** [comment]
@BC-tymurbiedukhin 
now it is truncated. in case of just removing overflow horizontal scroll will be permanent

https://user-images.githubusercontent.com/66325265/110444991-543b8980-80c6-11eb-8aba-50aeaf51c781.mov

after fix it will appear only in case of hover on specific swatch item

https://user-images.githubusercontent.com/66325265/110445099-77fecf80-80c6-11eb-90aa-7a3b4ffeec1b.mov

**Comment by BC-tymurbiedukhin** [comment]
@bc-as Could we move on with current look or hover image should be placed inside the screen?

**Pull request merged**